.github/workflows/node.js.yml

```diff
@@ -0,0 +1,25 @@
+name: Run application
+
+on:
+  push:
+    branches: [main]
```

Comment:
We can set the action to run on a schedule here - https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#onschedule

I guess we'd probably want to run this every Wednesday

Reply:
Here's an example of a script we have that runs on a schedule https://github.com/dxw/huntaway/blob/master/.github/workflows/assign.yml